### PR TITLE
EW-306 Add option to migrate internal users to newly configured LDAP

### PR DESCRIPTION
**Pull request by psachmann**
# Short Description

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

**Review by psachmann — COMMENTED** [review]
Additional question:
- Is the German translation enough or do we need at least an English translation as fallback?

File: src/store/import-users.ts [hunk 1]
```diff
@@ -91,12 +91,33 @@ export default class ImportUsersModule extends VuexModule {
...
+	@Mutation
+	deleteMatchMutation(importUserId: string): void {
+		const editedUser = this.importUserList.data.find(
+			(importUser) => importUser.importUserId === importUserId
+		);
+		if (editedUser) {
+			editedUser.match = undefined;
+			this.importUserList = { ...this.importUserList };
+		}
+	}
```
[the remark this answers is not in the document]
Comment (pkleybolte) [reply src/store/import-users.ts:103]: done

File: src/store/import-users.ts [hunk 2]
```diff
@@ -91,12 +91,33 @@ export default class ImportUsersModule extends VuexModule {
...
+	@Mutation
+	setUserFlagged(payload: { importUserId: string; flagged: boolean }) {
+		const editedUser = this.importUserList.data.find(
+			(importUser) => importUser.importUserId === payload.importUserId
+		);
+		if (editedUser) {
+			editedUser.flagged = payload.flagged;
+			this.importUserList = { ...this.importUserList };
+		}
+	}
```
[the remark this answers is not in the document]
Comment (pkleybolte) [reply src/store/import-users.ts:119]: done

**Review by psachmann — COMMENTED** [review]
In `de.json` are some words with gendering, wich might cause some issues with screen readers.

File: locale/de.json [hunk 3]
```diff
@@ -287,6 +287,10 @@
...
+	"pages.administration.ldap.activate.migrateExistingUsers.error": "Es ist ein Fehler aufgetreten. Der Migrations-Assistenten konnte nicht aktiviert werden. Das LDAP-System ist daher ebenfalls nicht aktiv geschaltet worden. Bitte probieren Sie es erneut. Falls das Problem bestehen bleibt, wenden Sie sich bitte an den Support",
+	"pages.administration.ldap.activate.migrateExistingUsers.info": "Wenn Sie schon Nutzer manuell angelegt haben, die zukünftig über das LDAP verwaltet werden sollen, dann können Sie den Migrations-Assistenten verwenden.<br> Durch den Assistenten werden LDAP-Nutzer mit bestehenden Nutzern verknüpft. Sie können die Zuordnung anpassen und müssen sie bestätigen, bevor die Zuordnung aktiv wird. Erst danach können sich Nutzer über das LDAP anmelden. <br>Wenn das LDAP-System ohne Migrations-Assistenten aktiviert wird, werden alle Nutzer aus dem LDAP importiert. Man kann die Nutzer nicht mehr nachträglich verküpfen.",
+	"pages.administration.ldap.activate.migrateExistingUsers.title": "Bestandsnutzer migrieren",
```
[the remark this answers is not in the document]
Comment (pkleybolte) [reply locale/de.json:293]: done

**Review by virgilchiriac — APPROVED** [review]

**Pull request merged**